platform_umbrella/apps/verify/test/postgres_test.exs

```diff
@@ -2,25 +2,26 @@ defmodule Verify.PostgresTest do
   use Verify.TestCase, async: false
 
   @tag :cluster_test
-  test "Can start a postgres cluster", %{session: session, control_url: url} do
+  test "can start a postgres cluster", %{session: session, control_url: url} do
     cluster_name = "int-test-#{:rand.uniform(10_000)}"
 
     session
     |> visit(url <> "/postgres/new")
     |> assert_has(Query.text("New Postgres Cluster"))
-    |> click(Query.text_field("cluster[name]"))
     |> fill_in(Query.text_field("cluster[name]"), with: cluster_name)
```

Comment:
Yeah, this really doesn't work for us. Not sure what's going on. Going to try to get a small repro to see if we can get it fixed.

Reply:
As long as fill_in works, no big deal.